### PR TITLE
I added rat as a food.

### PR DESCRIPTION
**Pull request by moosahassanx**
The rat is raw.

### CODE REVIEW

File: Tasks.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ On GitKraken, right click on the latest `main` branch commit, and select create
 
 6. Stage and commit your changes to that file. 
 
-7. Push your commits to `main`. This saves your commits to a branch on the remote repo, which is in this case on GitHub.
+7. Push your commits to your repo. This saves your commits to a branch on the remote repo, which is in this case on GitHub.
```
Comment (Bidski) [inline Tasks.md:32]: While I applaud you trying to fix @KipHamiltons horrendous typos, this needs to be done in a separate PR.

File: Foods.yaml [hunk 2]
```diff
@@ -6,5 +6,5 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["Pizza"]
+  - ["Pizza", "Rat", "Sewage water"]
```
Comment (Bidski) [inline Foods.yaml:9]: I do not approve of this

**Review by Bidski — CHANGES_REQUESTED** [review]

File: Foods.yaml (hunk 2 above)
Comment (LachlanCourt) [inline Foods.yaml:9]: Rat is not food :vomiting_face:

**Review by LachlanCourt — CHANGES_REQUESTED** [review]

File: Foods.yaml (hunk 2 above)
Comment (KipHamiltons) [inline Foods.yaml:9]: It was pointed out to me yesterday that we should try to stick to `snake_case` for `yaml` files in the NUbots codebase
```suggestion
  - ["pizza", "rat", "sewage_water"]
```

**Review by KipHamiltons — COMMENTED** [review]
It's worth going through the process of dealing with the merge conflicts when you try to merge the `main` branch in to this branch

File: Tasks.md (hunk 1 above)
Comment (BryceTuppurainen) [reply Tasks.md:32]: Liam and I fixed this up in PR #15

File: Foods.yaml (hunk 2 above)
Comment (BryceTuppurainen) [reply Foods.yaml:9]: Do we have an agreed upon set of standards for these yet? Otherwise, that would be quite useful.
